rust/src/applayer.rs

```diff
@@ -155,6 +159,7 @@ impl AppLayerTxData {
             file_flags: 0,
             file_tx: 0,
             stream_logged: 0,
+            updated: [true; 2],
```

Comment:
creating a new tx should not lead to it being updated in both dirs? Just in the dir it was created in?

Reply:
right, will optimize